app/views/coronavirus_form/session_expired.html.erb

```diff
@@ -1,27 +1,17 @@
-<%# GOVWALES: :back_link addition. Lines 1-8 %>
-<% content_for :back_link do %>
-  <%= render "govuk_publishing_components/components/back_link", {
-    text: t("govwales_breadcrumbs.start_again_text"),
-    href: t("govwales_breadcrumbs.start_again_url"),
+<% content_for :title do %><%= t("session_expired.title") %><% end %>
+  <% content_for :meta_tags do %>
+    <meta name="description" content="<%= t("session_expired.title") %>" />
+  <% end %>
+  
+  <%= render "govuk_publishing_components/components/title", {
+    title:t("session_expired.title"),
+    margin_top: 0
+  } %>
+  
+  <%= sanitize(t("session_expired.body_text")) %>
+  
+  <%= render "govuk_publishing_components/components/button", {
+    text: "Start now",
```

Comment:
Translation

Reply:
This is the original GDS one, that we don't see. Could pick it up in a snag task?